README.md

```diff
@@ -66,6 +67,11 @@ class User < ActiveRecord::Base
       }
     ],
 
+    # 2): To use credentilas dynamically from database use lamda notation.
+    # ftp_credentials is class method here in which you can access
+    # related association to load credentials from database.
+    ftp_servers: -> (attachment) { attachment.instance.ftp_credentials }
```

Comment:
This line should be commented out to show that only one `:ftp_servers` key should be given in the options hash.

So overall I suggest this reduced description:
```ruby
# 2): Provide a proc or lambda that returns the ftp server(s), which takes the current attachment as an argument
# ftp_servers: -> (attachment) { attachment.instance.ftp_credentials }
```